### PR TITLE
LDAP Introduction

### PR DESCRIPTION
**Pull request by justin-stephenson**

### CODE REVIEW

File: src/docs/ldap/ldap-introduction.rst [hunk 1]
```diff
@@ -1,2 +1,29 @@
 Introduction to LDAP Integration
-################################
\ No newline at end of file
+################################
+
+The LDAP Protocol is a well known cross platform protocol implemented in various LDAP server software components in small to enterprise level environments. SSSD on Linux clients is able to consume LDAP information with LDAP servers on Linux clients using the traditional SSSD LDAP provider. SSSD can use LDAP information for identity and authentication operations.
+
+SSSD strives to to work well with multiple RFC-conforming (*rfc2307bis*, *rfc2307*) LDAP server implementations such as 389-ds-base, OpenLDAP, Windows Active Directory, and several others. LDAP provider features include*, not limited to
```
Comment (pbrezina) [inline src/docs/ldap/ldap-introduction.rst:6]: to to -> to

File: src/docs/ldap/ldap-introduction.rst [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+
+* SASL/SSL/TLS suport
+* LDAP Service discovery
+* Control search behavior with search base customization and nesting level 
+* Paging configurable options
+* LDAP Access filters
+* SUDO LDAP support
+* Password policy support
+
+\* client side support for features is dependent on server side support
```
Comment (pbrezina) [inline src/docs/ldap/ldap-introduction.rst:16]: convert  it to .. note?

**Review by pbrezina — CHANGES_REQUESTED** [review]
Minor comments inside.

**Comment by justin-stephenson** [comment]
See latest update.

**Pull request merged**